Assets/Scripts/Score/OverallScoreManager.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+using TMPro;
+
+public class OverallScoreManager : MonoBehaviour
+{
+    [SerializeField]
+    private TextMeshProUGUI highScore;
+
+    private float overallScore;
+    private int lastLevel = 2;
```

Comment:
Currently, this variable duplicates the `lastLevel` constant in `LevelEndTriggerController.cs`. I think it's better to add `Constants.cs` and use a public constant there in both this script and `LevelEndTriggerController.cs`.

Reply:
Agreed